benchbuild/utils/actions.py

```diff
@@ -154,8 +144,9 @@ class StepClass(abc.ABCMeta):
     def __new__(mcs, name, bases, namespace, **_):
         result = abc.ABCMeta.__new__(mcs, name, bases, dict(namespace))
 
-        NAME = result.NAME
-        DESCRIPTION = result.DESCRIPTION
+        NAME: str = result.NAME
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "NAME" doesn't conform to '[a-z_][a-z0-9_]{2,30}$' pattern](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)